extension/source/Controllers/Account/IAccountTrackerController.ts

```diff
@@ -0,0 +1,37 @@
+import { BaseConfig, BaseState, IController } from '../interfaces';
+
+export interface IAccountTrackerController<C, S> extends IController<C, S> {
+  /**
+   * Syncs accounts from preferences controller
+   */
+  syncAccounts(): void;
+
+  /**
+   * Refreshes the balances of all accounts
+   */
+  refresh(): Promise<void>;
+}
+
+export interface AccountTrackerConfig extends BaseConfig {
+  _currentBlock?: string;
```

Comment:
Why is the current block a configuration? Isn't that state?

Reply:
Added as a comment.
Config represents variables which subscriber doesn't care about.
State represents variables which subscriber is interested in